### PR TITLE
Fix issue/PR templates

### PR DESCRIPTION
**Pull request by ffaf1**
See: #9100.

There are some text wrapping issue, plus it seems to me that PR templates are not shown.

### CODE REVIEW

**Review by Kleidukos — APPROVED** [review]

**Review by geekosaur — APPROVED** [review]

**Comment by ffaf1** [comment]
Ok, apparently [there is a trick to have multiple PR templates on GitHub](https://stackoverflow.com/questions/73771068/multiple-templates-for-pull-requests-on-github), but I dislike it a lot. It involves the user having to click “Preview” and then the correct template.

So I made a single PR template where the user simply pick one of tho options and erases the other one. I will post the template as is below, in another comment. Let me know if you prefer the “Preview” option.

**Review by ulysses4ever — APPROVED** [review]

**Comment by ffaf1** [comment]
We may want to merge this fast, or for the next two days new PRs will be templateless! I'll let someone with seniority decide.

**Comment by ulysses4ever** [comment]
Absolutely. Will have to wait for CI to finish though.

**Pull request merged**
